about.html

```diff
@@ -0,0 +1,135 @@
+                    activists - we hope you'll
+                    consider joining us this year to build a stronger, more vibrant commons together
+                </p>
+
+                <div class="abt-contact">
+                    <p class="abt-contact-desc">Please contact us per Email for nay further questions about CC Global
+                        Summit 2023!</p>
+                    <p class="about-contact-email">omololuayk@gmail.com</p>
+                </div>
+            </div>
+
+        </div>
+    </header>
+    <section class="abt-logo-section">
+        <div class="desktop-abt-logo">
+            <div class="abt-logo-text-container">
+                <h4 class="abt-logo-text">Creative Content Global Summit 2023 Logo</h4>
+                <span class="line"></span>
+                <p class="logo-section-desc">The logo of CC Global Summit 2023 was decided through the logo competition
+                    from 5, June to 7, July.</p>
+                <p class="desktop-logo-desc">The logo of CC Global Summit 2015 was decided through the logo competition
+                    from 8, June to 7, July.
+                    After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal
+                    won the prize.
+                    After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as
+                    follows:</p>
+            </div>
+            <img src="images/about_logo_desktop.png" class="abt-desktop-img" alt="gloal summit logo">
+        </div>
+    </section>
+    <section class="abt-past-summit">
+        <div class="abt-past-summit-content">
+            <h4 class="abt-logo-text">See the past CC Global submits</h4>
+            <span class="line"></span>
+            <p class="abt-past-summit-desc">Take a look at the past two CC Global Summits which took place in Buenos
+                Aires and in Warsaw</p>
+        </div>
+        <div class="abt-past-summit-pic">
+            <div class="abt-past-pic-edit">
+                <img src="images/about_2011 1.png" alt="global summit picture">
+            </div>
+            <div class="abt-past-pic-edit">
+                <img src="images/about_2013 1.png" alt="global summit picture">
+            </div>
+        </div>
+    </section>
+
+    <section id="Partner" class="partner-container abt-partner-container">
+        <h5>Partner</h5>
+        <span class="line"></span>
+        <div class="partner-img">
+            <img src="images/mozilla 1.png" alt="mozilla logo">
+            <img src="images/googlekorea 1.png" alt="google logo">
+            <img src="images/naver 1.png" alt="naver logo">
+            <img src="images/daumkakaologo 1.png" alt="dumkakao logo">
+            <img src="images/airbnb 1.png" alt="airbnb logo">
+        </div>
+    </section>
+
+    <footer class="footer">
```

Comment:
- [ ] Your about footer for desktop device doesn't have the correct background color as can be seen in the project design provided to you 
- project design
![ab-ft](https://github.com/Ayokunnumi1/Conference-Page/assets/89055502/8af1ff22-c004-4442-ae5a-e60eb3bee583)